previous-projects.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar:
+  nav: "docs"
+permalink: /previous-projects.html
+layout: splash
+header:  
+    #image: /assets/images/bg.jpg
+    video: /assets/images/waves.mp4
+    overlay_color: "rgba(63, 108, 152, 1)"
+intro:
+  - excerpt: "Previous projects"
+---
+
+{% include feature_row id="intro" type="center" %}
+
+<style type="text/css" media="screen">
```

Comment:
I know this is just a prototype but it would be better to have the CSS in independent files.